src/component/footer/footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./footer.module.scss";
+
+const Footer = ({
+  handleClickBack,
+  handleClick,
+  previousPageLink,
+  nextPageLink,
+}) => {
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.footer__backoptions} onClick={handleClickBack}>
+        <div className={styles.footer__arrowleft} />
```

Comment:
If this is the fist content, we should now show the link. Only show the content "This is the first page".
Means this is not the link, wen can't click it.

And optimize the style to make it small size and beautiful.